### PR TITLE
TCS-006 header of story is in capital letters 

### PR DESCRIPTION
**Pull request by 4stuntman4**

### CODE REVIEW

File: data/testData.json [hunk 1]
```diff
@@ -14,5 +14,6 @@
     "Rags and Riches": 4,
     "Tragedy": 5,
     "Comedy": 6
-  }
-}
\ No newline at end of file
+  },
+  "headerStory": "Two Cats And A LadyBug"
```
Comment (max9019) [inline data/testData.json]: Move "headerStory" to expected.json please.

File: test/smoke/MLH-543_storytoRead_TCS.js [hunk 2]
```diff
@@ -6,34 +6,36 @@ const inputValues4AndClick = require('../../helpers/inputValues4AndClick');
 
 describe('MLH-543 Page header is present and correct  ', function() {
 
-    it('TCS-001 page header My little hero is present', function (){
+    xit('TCS-001 page header My little hero is present', function (){
```
Comment (max9019) [inline test/smoke/MLH-543_storytoRead_TCS.js]: what's up with "xit"?

File: test/smoke/MLH-543_storytoRead_TCS.js [hunk 3]
```diff
@@ -6,34 +6,36 @@ const inputValues4AndClick = require('../../helpers/inputValues4AndClick');
...
         browser.url('');
         const headerMLH = $(sel.headerMLH).isDisplayed();
         expect(headerMLH).toEqual(true);
     });
 
-    it('TCS-002 page header My little hero is correct', function (){
+    xit('TCS-002 page header My little hero is correct', function (){
```
Comment (max9019) [inline test/smoke/MLH-543_storytoRead_TCS.js]: what's up with "xit"?

File: test/smoke/MLH-543_storytoRead_TCS.js [hunk 4]
```diff
@@ -6,34 +6,36 @@ const inputValues4AndClick = require('../../helpers/inputValues4AndClick');
...
         expect(uploadedPicture).toContain('app/static');
     });
 
     it('TCS-006 the header of the story is in capital letters', function (){
-
-
+        browser.url('');
+        inputValues4AndClick(data.name,data.gender.she,data.age,data.storyType.Comedy);
+        const headerStory = $(sel.headerStory).getText();
+        expect(headerStory).toStrictEqual(data.headerStory);
```
Comment (max9019) [inline test/smoke/MLH-543_storytoRead_TCS.js]: If you are checking capital letters then why using toStrictEqual? If you're checking if the Name of the story is correct then toStrictEqual is OK but it gives a positive test while the name of the story is wrong "Two Cats And A Name 123"

**Review by max9019 — CHANGES_REQUESTED** [review]

File: data/testData.json (hunk 1 above)
Comment (4stuntman4) [reply data/testData.json]: moved

File: test/smoke/MLH-543_storytoRead_TCS.js (hunk 4 above)
Comment (4stuntman4) [reply test/smoke/MLH-543_storytoRead_TCS.js]: it does not matter - we check capital letters and not capital letters + 123. Perfectly works with  two Cats And A Name as well as other variants

File: test/smoke/MLH-543_storytoRead_TCS.js (hunk 2 above)
Comment (4stuntman4) [reply test/smoke/MLH-543_storytoRead_TCS.js]: my original branch does not have it

**Review by max9019 — APPROVED** [review]

**Pull request merged**